evennia/utils/tests/test_containers.py

```diff
@@ -63,19 +92,45 @@ def test_start_with_typeclassed_script_loads_it(self):
     self.assertEqual(gsc.typeclass_storage['script_name'], GoodScript)
 
   @override_settings(GLOBAL_SCRIPTS={'script_name': {'typeclass': 'evennia.utils.tests.test_containers.BadScript'}})
-  def test_start_with_bad_typeclassed_script_skips_it(self):
+  def test_start_with_bad_typeclassed_script_raises(self):
+    gsc = containers.GlobalScriptContainer()
+
+    with self.assertRaises(Exception) as cm:
+        gsc.start()
+        self.fail("An exception was expected but it didn't occur.")
+
+    self.assertEqual(len(gsc.typeclass_storage), 1)
+    self.assertIn('script_name', gsc.typeclass_storage)
```

Comment:
So `start()` raised but `gcs.typeclass_storage` has it as a "loadable" script. IMHO, this is a bad quirk.